src/index.ts

```diff
@@ -50,11 +56,25 @@ export const defaultOptions = {
 
     return false;
   },
+  useGETForHashedQueries: false,
 };
 
+function definitionIsMutation(d: DefinitionNode) {
+  return d.kind === 'OperationDefinition' && d.operation === 'mutation';
+}
+
+function operationIsQuery(operation: Operation) {
```

Comment:
this will pass for subscriptions too fyi. Since this link isn't designed to be used there its probably not an issue, but long term I think we do want Engine to be able to upgrade a connection to a subscription so it may come up

Reply:
Don't subscriptions need to be sent over websockets anyway? Engine doesn't really support subscriptions yet.